jax/_src/numpy/setops.py

```diff
@@ -830,6 +832,7 @@ def unique(ar: ArrayLike, return_index: bool = False, return_inverse: bool = Fal
     >>> print(counts)
     [2 1]
   """
+  del sorted  # unused
```

Comment:
We could probably save a step in the computation if we thread this through to `_unique_sorted_mask`.

If you just want to get this fix in now, maybe add a TODO for that?

Reply:
Good question! It's not totally clear to me how we would benefit from this since AFAICT, our implementation requires sorting. But, I've left a TODO, and it would be a great thing to investigate in the future!